src/components/Accordion/AccordionItem.tsx

```diff
@@ -52,30 +52,33 @@ const AccordionHeader = ({
 
   // Render all icons and use css to conditionally show/hide the correct one
   const lockIconComponent = (
-    <img
-      className={getIconClasses(baseClassName, isLargeVariation, 'lock')}
-      src={lockIcon}
-      data-testid={`${id}-lockedIcon`}
-      aria-hidden
-    />
+    <div>
+      <LockIcon
+        className={getIconClasses(baseClassName, isLargeVariation, 'lock')}
+        data-testid={`${id}-lockedIcon`}
+        aria-hidden
+      />
+    </div>
   );
 
   const plusIconComponent = (
-    <img
-      className={getIconClasses(baseClassName, isLargeVariation, 'plus')}
-      src={plusIcon}
-      data-testid={`${id}-plusIcon`}
-      aria-hidden
-    />
+    <div>
```

Comment:
Is there a reason we need this wrapped in divs with no classes? The previous img tag didn't have a div tag so I would think the svg  component would work the same

Reply:
yep, div is responsible for spacing